RegularApi.Tests/Fixtures/IDaoFixture.cs

```diff
@@ -0,0 +1,12 @@
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using RegularApi.Dao.Model;
+
+namespace RegularApi.Tests.Fixtures
+{
+    public interface IDaoFixture
```

Comment:
If you go this way, then you have to create a test for this Dao

Reply:
I don't think so, because is a fixture to use it with dealing with crud operations instead of create private methods in each dao and repeating in every place where we need, so, I though moved them in a fixture to reuse them.